RELEASE/scripts/autoscend.ash

```diff
@@ -3231,8 +3231,9 @@ boolean LX_steelOrgan()
 		{
 			foreach it in $items[Hilarious Comedy Prop, Victor\, the Insult Comic Hellhound Puppet, Observational Glasses]
 			{
-				if(item_amount(it) > 0)
+				if(possessEquipment(it))
 				{
+					equip(it);
```

Comment:
if the user did part of this manually, it's possible to be in a state where this equip will fail when equipping the puppet if no weapon is equipped, since you can't have an offhand with no weapon equipped

Reply:
You can have an offhand without a weapon. I just verified it.
But... this would fail if you have a 2 handed weapon, which autoForceEquip would handle. I will change it.
Also, boris would also fail. gotta adjust for that